packages/salesforcedx-vscode-apex/src/commands/forceGenerateFauxClasses.ts

```diff
@@ -190,6 +190,15 @@ export async function forceGenerateFauxClassesCreate(
 export async function initSObjectDefinitions(projectPath: string) {
   const hasDefaultUsernameSet =
     (await getDefaultUsernameOrAlias()) !== undefined;
+  initSObjectDefinitionsWithUserName(projectPath, hasDefaultUsernameSet).catch(e =>
+    telemetryService.sendErrorEvent({
+      message: e.message,
+      stack: e.stack
+    })
+  );
+}
+
+export async function initSObjectDefinitionsWithUserName(projectPath: string, hasDefaultUsernameSet: boolean) {
```

Comment:
`initSObjectDefinitions` and `initSObjectDefinitionsWithUserName` should be merged into one method, we don't have a scenario where we want to trigger a refresh operation when the project does not have a default username assigned.